src/Storefront/Resources/views/storefront/component/address/address-manager-modal.html.twig

```diff
@@ -0,0 +1,133 @@
+<div class="js-pseudo-modal-template-root-element address-manager-modal">
+    <div class="modal-header align-items-start">
+        {% block address_manager_modal_title %}
+        <h1>
+            {{ 'account.addressModalTitle'|trans|sw_sanitize }}
+        </h1>
+        {% endblock %}
+
+        {% block address_manager_modal_close_button %}
+            <button type="button" class="btn-close" data-bs-dismiss="modal" aria-label="Close"></button>
+        {% endblock %}
+    </div>
+    <div class="modal-body">
+        {% block address_manager_modal_tab_group %}
+            <div class="btn-group address-manager-modal-tab-navigation-list w-100 sticky-top bg-white"
+                 id="address-manager-modal-tab-navigation-tabs">
+                {% block address_manager_modal_navigation_shipping %}
+                    <input type="radio"
+                           class="btn-check"
+                           name="btnradio"
+                           id="shipping-address-tab"
+                           autocomplete="off"
+                           checked>
```

Comment:
Please check all elements here and correct formatting of line breaks in attributes.